hooks/gen_docs/utils.py

```diff
@@ -28,7 +28,7 @@ def copy_file(
             shutil.copyfileobj(file_source, file_dest)
             return
         if header is None:
-            header_encoded = f"<!-- This file is autogenerated by {Path(inspect.stack()[1].filename ).relative_to(ROOT)}. -->\n\n".encode()
+            header_encoded = f"<!-- This file is autogenerated by {Path(inspect.stack()[1].filename ).relative_to(ROOT)}. -->\n".encode()
```

Comment:
what's with the extra space? does Ruff format it like this?
```suggestion
            header_encoded = f"<!-- This file is autogenerated by {Path(inspect.stack()[1].filename).relative_to(ROOT)}. -->\n".encode()
```

Reply:
Likely a missclick, it's not `ruff`